src/views/Home/Extraction/index.tsx

```diff
@@ -291,88 +268,109 @@ function Extraction() {
         });
     }, []);
 
+    /*
     const handleSelectAllChange = useCallback((checked: boolean) => {
         if (!extractionsResponse?.extractions?.results) return;
         const currentPageIds = extractionsResponse.extractions.results.map((item) => item.id);
         setSelectedIds(checked ? currentPageIds : []);
     }, [extractionsResponse]);
+    */
```

Comment:
why was this comment out?

Reply:
Not handling it yet. Will come in another commit.